Cargo.toml

```diff
@@ -121,6 +121,7 @@ tokio-threadpool = "0.1.13"
 vlog = "0.1.4"
 twoway = "0.2.0"
 ordered-float = "1.0"
+nom = "5.0.0-alpha2"
```

Comment:
If the latest version `5.0.0-beta1` better?

Reply:
Yes! `5.0.0-alpha2` is better! However, there are some breaking changes in `alpha2`, I am still working on it.